src/components/Container/Container.tsx

```diff
@@ -37,15 +37,19 @@ export interface ContainerProps<T extends ElementType> {
   orientation?: Orientation;
   padding?: PaddingOptions;
   wrap?: WrapOptions;
+  height?: string;
+  maxHeight?: string;
+  minHeight?: string;
+  overflow?: string;
 }
 
 const Container = forwardRef(
   <T extends ElementType = "div">(
     {
       component,
-      alignItems = "center",
+      alignItems,
       children,
-      fillWidth = false,
+      fillWidth = true,
```

Comment:
If we change this default it will break the UI. We need to mark this as a breaking change and update all of SQL-Console UI before updating the ClickUI version there.

Reply:
Got it
I'll update it in sql-console

Reply:
https://github.com/ClickHouse/control-plane/pull/6702